packages/api-headless-cms/src/crud/contentEntry.crud.ts

```diff
@@ -826,7 +826,7 @@ export const createContentEntryCrud = (params: CreateContentEntryCrudParams): Cm
             version: nextVersion,
             savedOn: getDate(rawInput.savedOn, currentDate),
             createdOn: getDate(rawInput.createdOn, currentDate),
-            publishedOn: getDate(rawInput.publishedOn, originalEntry.publishedOn),
+            publishedOn: getDate(rawInput.publishedOn, undefined),
```

Comment:
Just got pinged by @Pavel910 about this thread here.

I suggest we don't do anything here @neatbyte-vnobis, because this is currently being reworked in my [PR](https://github.com/webiny/webiny-js/blob/a10d00f09c7ed245b57253a2533117c8a4a22e19/packages/api-headless-cms/src/crud/contentEntry.crud.ts#L876-L930).

We'll basically have revision and entry-level sets of these meta fields. For revision-level, yes, the published-related fields will be unset. But, for entry-level, fields values will be propagated throughout all of the revisions.

So, let's undo this, and we'll